packages/dapp/src/components/Dashboard/Dashboard.tsx

```diff
@@ -14,9 +14,12 @@ import DashboardActivity from "./DashboardActivity";
 
 const StyledDashboardActivityContainer = styled.div`
   box-sizing: border-box;
-  padding: 0 0 200px 396px;
-  margin: -550px auto 0;
-  width: 1200px;
+  padding: 0 0 0px 0px;
+  margin-top: 40px;
+  ${mediaQueries.MOBILE} {
+    max-width: 100%;
+    margin-top: 0px;
```

Comment:
Padding can just be `padding: 0;` or you might not even need padding at all unless it's inheriting it from somewhere else. 

You can remove px in margin-top: `margin-top: 0;`.